doc/conf.py

```diff
@@ -87,7 +87,9 @@
 exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
 
 # The name of the Pygments (syntax highlighting) style to use.
-pygments_style = 'sphinx'
+# for the theme "sphinx_book_theme", `pygments_style` has no effect
+# on the syntax highlight theme. Instead, modify it with `html_theme_options`
+# pygments_style = 'sphinx'
```

Comment:
maybe remove all these comments? Not sure why we would keep them.

Reply:
No strong opinion.

Indeed, I was less sure about adding this comment.